packages/maintenance-app/backend/src/scripts/utils/run.ts

```diff
@@ -2,19 +2,31 @@ import * as path from 'path'
 import config from '../../utils/config'
 import { execFile } from 'child_process'
 
-export default function run(script: string): Promise<string> {
+export type EnvironmentVariables = {
+  [name: string]: boolean | string | number
+}
+
+export default function run(script: string, env?: EnvironmentVariables): Promise<string> {
+  // need to set the current path properly before starting a script
   const pathToScript: string = path.normalize(config.scriptLocation + script)
+  const options = env ? { env: { PATH: process.env.PATH, ...env } } : null
```

Comment:
`PATH` is needed to call globally accessible scripts on the host machine (ie docker)